engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/KafkaAvroSourceFactory.scala

```diff
@@ -47,20 +51,33 @@ class KafkaTypedAvroSourceFactory[T: TypeInformation](schemaRegistryProvider: Sc
              )
              @NotBlank
```

Comment:
do we want to configure editors also here?

Reply:
You're right. I'v added String editor. But in future would be nice to have AvroEditor => AceEditor with mode JSON ;)